src/constants.ts

```diff
@@ -11,6 +11,7 @@ export const managedEnvironmentsId = 'managedEnvironments';
 export const containerAppsId = 'containerApps';
 export const appProvider: string = 'Microsoft.App';
 export const webProvider: string = 'Microsoft.Web';
+export const registryProvider: string = 'Microsoft.ContainerRegistry';
```

Comment:
I think we should be good to move all these consts to `getVerifyProvidersStep`

Reply:
We can move them there, but they won't be localized to that file so will still have to export a few of them

Reply:
Oh are they? Never mind then-- I'm surprised we are using them anywhere else.

Reply:
two are, two aren't :P